x/profiles/keeper/alias_functions_test.go

```diff
@@ -97,6 +97,39 @@ func (suite *KeeperTestSuite) TestKeeper_GetProfiles() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestKeeper_IterateUserIncomingDTagTransferRequests() {
+	address := "cosmos19xz3mrvzvp9ymgmudhpukucg6668l5haakh04x"
+	requests := []types.DTagTransferRequest{
+		types.NewDTagTransferRequest(
+			"DTag1",
+			"cosmos10nsdxxdvy9qka3zv0lzw8z9cnu6kanld8jh773",
+			address,
+		),
+		types.NewDTagTransferRequest(
+			"DTag1",
+			"cosmos1xcy3els9ua75kdm783c3qu0rfa2eplesldfevn",
+			address,
+		),
+		types.NewDTagTransferRequest(
+			"DTag1",
+			address,
+			"cosmos10nsdxxdvy9qka3zv0lzw8z9cnu6kanld8jh773",
+		),
+	}
+
+	for _, request := range requests {
+		err := suite.k.SaveDTagTransferRequest(suite.ctx, request)
+		suite.Require().NoError(err)
+	}
+
+	iterations := 0
+	suite.k.IterateUserIncomingDTagTransferRequests(suite.ctx, address, func(index int64, request types.DTagTransferRequest) (stop bool) {
+		iterations += 1
+		return index == 1
```

Comment:
Could it simply return false, instead?

Reply:
Fixed the test to make sure it stops earlier

Reply:
It seems that the test would be passed if the actual result is over 2 cause it would be stopped earlier on 2.

Reply:
@dadamu Yeah I've made it so to make sure that both cases of the `if !stop` are tested: when `stop == true` and when `stop == false`. The purpose of the test is to make sure that it iterates over only 2 out of the 3 transfer requests accordingly

Reply:
@RiccardoM I got it, it is used for testing stop quickly. Thanks for the answer!